### PR TITLE
Add runtime sub module

### LINKED ISSUE #19: [Runtime] Create apollo runtime module
For stuff that will be needed while running the app (eg.: executing queries, etc.)

### PR DESCRIPTION
**Pull request by sav007**
@felipecsl @martijnwalraven pls take a look

Resolves #19

### CODE REVIEW

**Comment by marwanad** [comment]
Adding
```  
licenses:
    - android-sdk-license-c81a61d9
```
 or `android-sdk-license-.+` under android in travis should fix the build.

Other than that LGTM.

**Comment by sav007** [comment]
@marwanad tried different combinations, still failing

**Comment by marwanad** [comment]
Looked at the travis logs and found this:

```
Error: Ignoring unknown package filter 'build-tools-24.0.2'
Warning: The package filter removed all packages. There is nothing to install.
         Please consider trying to update again without a package filter.
```
Apparently, it's trying to fetch those from `https://dl-ssl.google.com/android/repository/repository-10.xml` while it should be using `repository-11.xml`.

This works for me:
```
android:
  components:
    - tools
    - platform-tools
    - build-tools-24.0.2
    - android-24
  licenses:
    - 'android-sdk-license-.+'
```
I assume this would always install the latest build tools.

File: apollo-runtime/build.gradle [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+dependencies {
+    compile dep.jsr305
+}
+
+configurations {
+    jar
+}
+android.libraryVariants.all { variant ->
```
Comment (felipecsl) [inline apollo-runtime/build.gradle]: do you actually need this?

**Comment by felipecsl** [comment]
@sav007 I've never really seen that `licenses` thing added to travis.yml before. Did you try without it?

**Comment by sav007** [comment]
@felipecsl I tried first time and it failed but will try again later

File: apollo-runtime/build.gradle (hunk 1 above)
Comment (sav007) [inline apollo-runtime/build.gradle]: confess, just copied from delight, will review

**Comment by sav007** [comment]
@felipecsl @marwanad finally fixed travis, so this PR is ready to go pls review again

**Pull request merged**